src/shared-api-objects/current-patient.ts

```diff
@@ -4,17 +4,26 @@ import { mergeAll, filter, map } from "rxjs/operators";
 import { FetchResponse } from "../openmrs-fetch";
 
 let currentPatientUuid;
+const currentPatientUuidSubject = new ReplaySubject<PatientUuid>(1);
 const currentPatientSubject = new ReplaySubject<
   Promise<{ data: fhir.Patient }>
 >(1);
 
 window.addEventListener("single-spa:routing-event", () => {
   const u = getPatientUuidFromUrl();
-  if (u && currentPatientUuid !== u) {
+
+  if (u !== currentPatientUuid) {
     currentPatientUuid = u;
-    currentPatientSubject.next(
-      fhir.read<fhir.Patient>({ type: "Patient", patient: currentPatientUuid })
-    );
+    currentPatientUuidSubject.next(u);
+
+    if (u) {
+      currentPatientSubject.next(
+        fhir.read<fhir.Patient>({
+          type: "Patient",
+          patient: currentPatientUuid
```

Comment:
@joeldenning  why are you passing in `currentPatientUuid` as `patient` here?

Reply:
Because that's how fhir.js requires you to pass in the patientUuid. See related https://github.com/FHIR/fhir.js#get-resource and https://github.com/FHIR/fhir.js/pull/147